packages-content-model/roosterjs-content-model-core/lib/override/pasteCopyBlockEntityParser.ts

```diff
@@ -0,0 +1,37 @@
+import { isElementOfType, isNodeOfType } from 'roosterjs-content-model-dom';
```

Comment:
@JiuqingSong added this file to handle the issue when copying and Browser removing the percentage width. Do you think this is a good approach?